engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/history/UserOperationLogRestService.java

```diff
@@ -48,4 +51,15 @@
   List<UserOperationLogEntryDto> queryUserOperationEntries(@Context UriInfo uriInfo,
                                                            @QueryParam("firstResult") Integer firstResult,
                                                            @QueryParam("maxResults") Integer maxResults);
+
+  @PUT
+  @Path("/{operationId}/set-annotation")
+  @Produces(MediaType.APPLICATION_JSON)
```

Comment:
* let's return a `Response` object instead of `void`, so we're able to change what we return but stay backwards-compatible
* we should add a `Consumes` annotation here to be consistent with the other services